src/subprocess_test.cc

```diff
@@ -50,7 +51,13 @@ TEST_F(SubprocessTest, BadCommandStderr) {
     subprocs_.DoWork();
   }
 
-  EXPECT_EQ(ExitFailure, subproc->Finish());
+  ExitStatus exit = subproc->Finish();
+#ifdef _POSIX_VERSION
```

Comment:
Use `#else` on the _WIN32 branch instead.

Reply:
still missing

Reply:
I don't get what's missing. There's no `_WIN32` for `exit` anymore [here](https://github.com/ninja-build/ninja/blob/02a19024a9cefbbf1dfa7583e8dc0235895d1925/src/subprocess_test.cc#L55-L59) and [here](https://github.com/ninja-build/ninja/blob/02a19024a9cefbbf1dfa7583e8dc0235895d1925/src/subprocess_test.cc#L74-L78)

Reply:
Ah I was confused. But still wondering: Why is there a difference between `_POSIX_VERSION` and the rest?

Reply:
I don't know, but I would assume it's the `command not found` exit code for the win32. And POSIX returns 127 for it:

```
> command-not-found ; echo $?
zsh: command not found: command-not-found
127
```

Previously `subprocess-posix.cc` didn't propagate it, so any error was 1.

Reply:
I think we should just adjust the test that it checks != ExitSuccess then.

Reply:
No problem, will do in a moment